lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2107,15 +2098,6 @@ namespace Js
         JavascriptString* pThis = nullptr;
         GetThisStringArgument(args, scriptContext, L"String.prototype.toUpperCase", &pThis);
 
-        // Fast path for one character strings
```

Comment:
good to know. However just double check the history of when this was added and see if it was targeted for some specific benchmarks that are not in the suite.